hexa/pipelines/graphql/schema.graphql

```diff
@@ -77,12 +77,14 @@ enum PipelineError {
   PIPELINE_NOT_FOUND
   PIPELINE_VERSION_NOT_FOUND
   INVALID_CONFIG
+  PIPELINE_ALREADY_COMPLETED
 }
 
 extend type Query {
   pipelines(page: Int, perPage: Int): PipelinesPage!
   pipeline(id: String, name: String): Pipeline
   pipelineRun(id: String!): PipelineRun
+  pipelineRunCode(id: String): String
```

Comment:
This could be a simple `code` field on the `PipelineRun` type, or could be accessed though `run.pipeline.currentVersion.code`